src/panels/lovelace/cards/hui-history-graph-card.ts

```diff
@@ -133,20 +132,15 @@ export class HuiHistoryGraphCard extends LitElement implements LovelaceCard {
   private _setRedrawTimer() {
     // redraw the graph every minute to update the time axis
     clearInterval(this._interval);
-    this._interval = window.setInterval(() => this._redrawGraph(), 1000 * 60);
+    this._interval = window.setInterval(() => this._redrawGraph(), 1000 * 10);
```

Comment:
intentional to change this to 10 secs? Because comments doesnt match

Reply:
🙈 Oops